src/activeobject.h

```diff
@@ -66,7 +66,7 @@ enum ActiveObjectCommand {
 	AO_CMD_SET_BONE_POSITION,
 	AO_CMD_ATTACH_TO,
 	AO_CMD_SET_PHYSICS_OVERRIDE,
-	AO_CMD_UPDATE_NAMETAG_ATTRIBUTES,
+	AO_CMD_OBSOLETE1, // UPDATE_NAMETAG_ATTRIBUTES removed in 5.3.0
```

Comment:
```suggestion
	AO_CMD_OBSOLETE1, // UPDATE_NAMETAG_ATTRIBUTES deprecated in 0.5.0-dev
```

Reply:
Changed the comment to be more accurate. "Obsolete in 0.5.0-dev" is incorrect since those versions still understood this cmd.